db/system_keyspace.cc

```diff
@@ -954,6 +955,34 @@ schema_ptr system_keyspace::discovery() {
     return schema;
 }
 
+schema_ptr system_keyspace::events() {
+    static thread_local auto schema = [] {
+        auto id = generate_legacy_id(NAME, EVENTS);
+        auto b = schema_builder(NAME, EVENTS, std::make_optional(id))
+            .with_version(generate_schema_version(id))
+            .with_column("category", utf8_type, column_kind::partition_key)
```

Comment:
It's impossible to get a log of events with this, because it's sorted the wrong way.